examples/thttpd/thttpd_main.c

```diff
@@ -252,12 +259,18 @@ int main(int argc, FAR char *argv[])
 
   netlib_ifup("eth0");
 
+#ifdef CONFIG_THTTPD_NXFLAT
```

Comment:
why do we need guard romfs_img registery by CONFIG_THTTPD_NXFLAT?

Reply:
I first submitted the changes without the guard CONFIG_THTTPD_NXFLAT. The build broke with the error: unused variable 'desc'. This is the link for the build break: https://github.com/apache/incubator-nuttx-apps/runs/2581333837?check_suite_focus=true

Also, originally in the code there is a check to see if CONFIG_FS_ROMFS is enabled (line 91) if  CONFIG_THTTPD_NXFLAT is defined (line 86). For this reason I added the guard CONFIG_THTTPD_NXFLAT  before registering romdisk.